vars/daosPackagesVersion.groovy

```diff
@@ -60,7 +60,7 @@ String call(String distro, String next_version) {
         return version + dist
     }
 
-    if (target_branch.matches(testBranchRE())) {
+    if (target_branch =~ testBranchRE()) {
```

Comment:
Does `testBranchRE` not need an argument?
```
def call(String branch_type) {
...
        default:
            error "Unsupported branch type: ${branch_type}"
```

Reply:
As suggested by Brain, the original changes to `testBranchRE()` were moved to a new `branchTypeRE()` function.  So `testBranchRE()` no longer takes arguments and `branchTypeRE()` requires an argument.

Reply:
Ah, I misread the name :)